packages/manager/apps/telecom/src/app/telecom/pack/slots/informations/informations.html

```diff
@@ -229,6 +229,26 @@
                         </span>
                     </div>
                 </li>
+                <li class="oui-tile__item">
```

Comment:
I'm wondering if we are able (or not) to use:
https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-tile-webcomponent--with-buttons

Do you know if it's possible in this case?